src/main/resources/application.properties

```diff
@@ -0,0 +1,29 @@
+nasa.host = ${NASA_HOST:https://api.nasa.gov}
+nasa.apiKey = ${NASA_API_KEY:change_me}
+
+# H2 Database Configuration
+spring.datasource.url=${DATASOURCE_URL:jdbc:h2:file:~/testLOL}
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=${DATASOURCE_USERNAME:change_me}
+spring.datasource.password=${DATASOURCE_PASSWORD:change_me}
+
+# H2 Console Configuration (optional)
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
+
+# Create the schema (tables) and drop them when the application shuts down.
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
One of the conditions of the task was the creation of  schema and tables manually

Reply:
Good catch I missed this item. Thanks.